src/test/java/seedu/address/model/person/PersonTest.java

```diff
@@ -3,13 +3,7 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_DESCRIPTION_BOB;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_EMAIL_BOB;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_MAJOR_BOB;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_NAME_BOB;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_SM_GITHUB_BOB;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_SM_LINKEDIN_BOB;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_YEAR_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.*;
```

Comment:
I think the * import seems to be causing an issue with the checkstyle, might need to change it

Reply:
alright, just fixed it. thanks!